litmus-portal/graphql-server/manifests/cluster/3b_agents_deployment.yaml

```diff
@@ -21,6 +20,7 @@ metadata:
   name: agent-secret
   namespace: #{AGENT_NAMESPACE}
 stringData:
+  CUSTOM_TLS_CERT: "#{CUSTOM_TLS_CERT}"
```

Comment:
You will need to make changes in Chaoscenter directory, litmus-portal directory will be deprecated soon